src/main/java/com/olgaboiar/mint/Header.java

```diff
@@ -8,7 +8,7 @@
     private String blankLine;
     private String date;
     String headersToSend;
-    Set<String> allowedMethods;
+    String[] allowedMethods;
```

Comment:
Instead of using hardcoded strings, perhaps we can consider using Enum? Enum can help eliminate typos as well as being more explicit about the type of the object you're passing in.

Reply:
those are not hardcoded, but I've added enums for methods in https://github.com/olgaboiar/MintHTTPServer/pull/9/commits/331fa42dcd597f0e0b0678745fd6e7042e3ebd54